sdk/python/tests/integration/e2e/test_universal_e2e.py

```diff
@@ -78,7 +78,7 @@ def check_offline_and_online_features(
 def run_offline_online_store_consistency_test(
     fs: FeatureStore, fv: FeatureView
 ) -> None:
-    now = datetime.utcnow()
+    now = datetime.now()
```

Comment:
Why was this changed to `now` from `utcnow`?

Reply:
We have some places where we used utcnow and other places where we used now, which caused cases where we sometimes got no results when doing a historical retrieval